voucherify-service/src/integration/utils/mappers/buildValidationsValidateStackableParamsForVoucherify.ts

```diff
@@ -8,12 +8,14 @@ export function buildValidationsValidateStackableParamsForVoucherify(
 ) {
   return {
     // options?: StackableOptions;
-    redeemables: coupons.map((code) => {
-      return {
-        object: code.type ? code.type : 'voucher',
-        id: code.code,
-      };
-    }),
+    redeemables: coupons
+      .map((code) => {
+        return {
+          object: code.type ? code.type : 'voucher',
+          id: code.code,
+        };
+      })
+      .reverse(),
```

Comment:
fix root cause of the issue (order of coupons in custom filed) + describe in docs expected behaviour from the frontend perspective